Tests/CookiesTests.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System.Net.Http.Headers;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
+using Moq;
+using Cookies = UI.Pages.Cookies;
+
+
+
+namespace Tests;
+
+[Collection(nameof(SliceFixture))]
+public class CookiesTests : CleanSliceFixture
+{
+    public CookiesTests(SliceFixture fixture) : base(fixture)
+    {
+    }
+
+    [Theory]
+    [InlineData(true, true)]
+    [InlineData(null, true)]
+    [InlineData(true, null)]
+    [InlineData(null, null)]
+    public void Cookies_OnGet_Test(bool? consent, bool? preferencesSet)
```

Comment:
It's not clear from the test name or the asserts what this test is checking. It's possible the other tests in this file cover the full logic. If this is testing a specific path, consider renaming it and/or asserting (page) state rather than the result type

Reply:
Removed test individual tests now used